anyblok/tests/test_column.py

```diff
@@ -517,6 +517,8 @@ def test_datetime_by_query_filter(self):
         registry = self.init_registry(simple_column, ColumnType=DateTime)
         test = registry.Test.insert(col=now)
         Test = registry.Test
+        import ipdb
+        ipdb.set_trace()
```

Comment:
I don't feel you wanted to include it in you commit ;)

Reply:
ok forgot me I jump to a commit instead reading the whole diff

Reply:
it was a bad commit